chef/cookbooks/provisioner/templates/suse/crowbar_register.erb

```diff
@@ -193,6 +195,10 @@ PACKAGES_INSTALL="$PACKAGES_INSTALL ruby2.1-rubygem-libxml-ruby ruby2.1-rubygem-
 <% end -%>
 PACKAGES_INSTALL="$PACKAGES_INSTALL parted lvm2 ipmitool curl tack nfs-client rsyslog ntp tcpdump"
 
+case $ARCH in
+    x86_64) PACKAGES_INSTALL+=" biosdevname";;
```

Comment:
Is the package only available in x86_64, or is it not available on your arch but available on the other ones? Just wondering if this should be something like:
```shell
case $ARCH in
    ppc*) ;;
    *) PACKAGES_INSTALL+=" biosdevname";;
esac